Cargo.lock

```diff
@@ -0,0 +1,5 @@
+# This file is automatically @generated by Cargo.
```

Comment:
We might not need to commit this depending on the decision to include the binary in this repository. I would leave it out for now.

Reply:
good idea

Reply:
removed